string facial_expression
string text
button[] buttons
---
bool success